src/cli/cli.cpp

```diff
@@ -7913,7 +7916,8 @@ void Interpreter::HandleDiagnosticGetResponse(otError                 aError,
         {
         case OT_NETWORK_DIAGNOSTIC_TLV_EXT_ADDRESS:
             OutputFormat("Ext Address: '");
-            OutputExtAddressLine(diagTlv.mData.mExtAddress);
+            OutputExtAddress(diagTlv.mData.mExtAddress);
+            OutputLine("'");
```

Comment:
Why change the format to add extra `'`?

This may cause backward compatibility? Suggest not changing it unless good reason for it.

Reply:
The change here is to match the output ` Ext Address: '0e336e1c41494e1c'` in L7783. 
I am fine either way - keep the `''` or remove it, any preference?

Reply:
Ah! Interesting that doc is different from code.  I would favor for us to update the documentation and keep code same to keep for it to be backward compatible and consistent. I am not sure if we use `' '` for ext-address anywhere else in `cli`?

Reply:
Only here using single quote for ext address in cli

It might be weird if output extend address with a single quote like this `'0e336e1c41494e1c`, here minimally update the doc and code to remove the single quote for consistency in new patch